docs/guides/Integration-Guides/data-privacy/profile-system.md

```diff
@@ -4,24 +4,23 @@ slug: "profile-system"
 hidden: true
 createdAt: "2022-02-22T22:28:50.366Z"
 updatedAt: "2022-08-19T18:42:00.613Z"
-seeAlso: - "/docs/guides/changes-and-limitations-pii-data-architecture"
 ---
->❗ This feature is in closed beta phase, meaning we are working to improve it. Do not share this documentation with people outside of your company.
+>❗ This feature is in closed beta phase, meaning we are evolving it to support all scenarios. Do not share this documentation with people outside of your company.
 
-The [**Profile System**](https://developers.vtex.com/docs/guides/profile-system) is VTEX's single source of truth regarding shoppers' profile data. Other modules, such as **Checkout** and **Order Management**, can request data from the **Profile System** when necessary.
+The **Profile System** is VTEX's single source of truth regarding shoppers' profile data. Other modules, such as **Checkout** and **Order Management**, can request data from the **Profile System** when necessary.
 
 The Profile System API enables stores to:
 
-- Communicate with a single source of truth for shoppers’ data.
+- Communicate with a single source of truth for shoppers' data.
 - Safely store [PII](https://developers.vtex.com/docs/guides/pii-data-architecture) and sensitive information.
 
-Below you can learn more about some of the Profile System’s features and how to integrate with the APIs.
+Below you can learn more about some of the Profile System's features and how to integrate with the APIs.
 
 >⚠️ Data stored in the new Profile System is not related to documents saved in [Master Data v1](https://developers.vtex.com/docs/api-reference/masterdata-api) or [Master Data v2](https://developers.vtex.com/docs/api-reference/master-data-api-v2). The API endpoints and features described in this article only apply to documents registered in the new Profile System designed to handle PII data architecture.
 
 >❗ When implementing your integration:
 >
-> Do not forget to check the [Adaptations and limitations](https://developers.vtex.com/docs/guides/changes-and-limitations-pii-data-architecture) that impact other integrations.
+> Do not forget to check the [changes in VTEX features behavior to handle PII data](https://developers.vtex.com/docs/guides/changes-in-vtex-features-behavior-to-handle-pii-data) that impact other integrations, as well as the [current limitations related to PII data architecture](https://developers.vtex.com/docs/guides/limitations-of-the-pii-data-architecture-during-closed-beta).
```

Comment:
O que acha de transformar esse callout em uma seção "Before you start" e talvez colocar esses dois itens em formato de lista? Já tem muitos callouts na doc e talvez fique menos "assustador" do que um callout de danger. Também acho que dá uma sensação melhor da sequência do conteúdo